<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Ticket Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Admin User</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>611</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../icon/ticket_background.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_datetime">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CURRENT TIME</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionList_Users"/>
    <addaction name="actionUser_Payments_Info"/>
    <addaction name="actionTicket_info"/>
    <addaction name="actionAbout"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionList_Users">
   <property name="text">
    <string>List of Users</string>
   </property>
  </action>
  <action name="actionAbout_Us">
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
  <action name="actionTicket_info">
   <property name="text">
    <string>Ticket Info</string>
   </property>
  </action>
  <action name="actionUser_Payments_Info">
   <property name="text">
    <string>User Payments Info</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Software</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
